stage1/init/init.go

```diff
@@ -342,7 +342,9 @@ func getArgsEnv(p *Pod, flavor string, systemdStage1Version string, debug bool)
 	args = append(args, "--default-standard-output=tty") // redirect all service logs straight to tty
 	if !debug {
 		args = append(args, "--log-target=null") // silence systemd output inside pod
-		args = append(args, "--show-status=0")   // silence systemd initialization status output
+		// TODO remove this when we update stage1 to systemd v222
```

Comment:
remove 1 line or 2 lines? I suggest "// TODO remove --log-level=warning when we ..."